standalone/src/widget.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {
+    sendDesktopEvent,
+    playSound,
+} from 'plugin/utils';
+
+import {
+    logDebug,
+    logErr,
+} from 'plugin/log';
+
+import CallWidget from 'plugin/components/call_widget';
+
+import init from './init';
+
+init('widget', (store, theme) => {
```

Comment:
cool, the init and close cbs are a nice touch. Is there a functional reason for them, or is it mostly for organization?

Reply:
There is a reason since I needed to customize the logic further depending on what standalone client we are rendering. It will become clearer in the recordings PR which will make use of those in a slightly different way (plus more stuff as well). So the callbacks will actually move into a single `InitConfig` argument with various options to extend the logic.